### DISCUSSION TITLE
can you check your WEBCAM

### DISCUSSION THREAD

**Discussion by pit5bul** [stevedat/HP-Elitebook-830-G6-Hackintosh#10] [Ideas]
Can you please paste the Vedor id and product id from system information ?
Also .. can you attach a IOREG caputure.. im trying to see what i got different from your setup

**Comment by stevedat** [comment]
> Can you please paste the Vedor id and product id from system information ?
> Also .. can you attach a IOREG caputure.. im trying to see what i got different from your setup

It would not solve the puzzle. In case you use your camera for daily activity and sticky with the hackintosh, better use other option like usb camera connected, but the need is low is the usage, you have to live without it.

**Comment by pit5bul** [comment]
just want to check if same vendor id and same product id mate, before i
swap the webcam.

**Reply by stevedat** [reply]
It's same it should work, i didnot touch it, it works out of the box.
